packages/browser/src/lib/on-page-change.ts

```diff
@@ -7,23 +7,23 @@
  *
  * adapted from https://stackoverflow.com/questions/3239834/window-onbeforeunload-not-working-on-the-ipad/52864508#52864508,
  */
-export const onPageLeave = (cb: (...args: unknown[]) => void) => {
+export const onPageChange = (cb: (...args: boolean[]) => void) => {
```

Comment:
Nit:
This sig says that there are multiple bool args.
Type signature should just be (unloaded: boolean) => void